CMakeLists.txt

```diff
@@ -678,11 +678,6 @@ else()
 	target_compile_definitions(torrent-rasterbar PUBLIC TORRENT_DISABLE_DHT)
 endif()
 
-# Boost
-find_public_dependency(Boost REQUIRED COMPONENTS system)
-target_include_directories(torrent-rasterbar PUBLIC ${Boost_INCLUDE_DIRS})
-target_link_libraries(torrent-rasterbar PUBLIC ${Boost_SYSTEM_LIBRARY})
```

Comment:
if someone is on a system that has boost.system as a compiled library, removing this will cause link errors, right? Is it important to remove this?
A system where boost system is used as header-only, presumably cmake would still create a target for it, just not link against a library.

Reply:
Thank you. I didn't got that (Boost REQUIRED COMPONENTS system) is a boost detector.